src/web/core/src/main/java/org/geoserver/web/data/resource/InternationalStringPanel.java

```diff
@@ -301,21 +300,19 @@ private boolean i18nVisible() {
         public void validate(IValidatable<Locale> iValidatable) {
             Locale locale = iValidatable.getValue();
             List<GrowableStringModel.InternationalStringEntry> items = provider.getItems();
-            if (locale != null) {
-                long count =
-                        items.stream()
-                                .filter(i -> i.getLocale() != null && i.getLocale().equals(locale))
-                                .count();
-                if (count >= 1 && iValidatable.getModel().getObject() == null) {
-                    String message =
-                            new StringResourceModel(
-                                            "InternationalStringPanel.duplicatedLocale",
-                                            InternationalStringPanel.this)
-                                    .getString();
-                    ValidationError error = new ValidationError();
-                    error.setMessage(message);
-                    iValidatable.error(error);
-                }
+            long count =
```

Comment:
Here should not be a matter of simply remove the if condition, but the method should be validation also that there are no duplicated entries with null Locale. Once this is fixed I guess that the changes made in the related geotools pr made to GrowableInternationalString add method can be removed, leaving just the copy constructor fix.